config/swagger.yaml

```diff
@@ -15,31 +15,26 @@ schemes:
   - http
 paths:
   /ping:
-    get:
+    post:
       tags:
         - external
       operationId: Ping
       description: ''
       produces:
         - application/json
+      parameters:
+        - in: body
+          name: body
+          description: Ping a peer
+          required: true
+          schema:
+            $ref: '#/definitions/Ping'
       responses:
         '200':
           description: Sample response
           schema:
             $ref: '#/definitions/Ping'
```

Comment:
Shouldn't we have a 404 error response, too?

Reply:
Indeed